setup/setup_blocks.js

```diff
@@ -24,7 +24,7 @@ let ProvisionBlocks = (api, userId, key, callback = function() {}, status = func
           block_id: block.id,
           type: 'js',
           event: 'js-after-presence',
-          channels: 'chat-engine-global',
+          channels: '*',
```

Comment:
We'll run this on every channel to avoid confusion. If the cost is high, we can direct customers to change it.

Reply:
We can avoid this by manually storing state via http request endpoint to /server. This is dumb because we avoid pubnub, but will prevent us from running this function when unnecessary.